R/python.R

```diff
@@ -760,6 +783,8 @@ py_set_attr <- function(x, name, value) {
 #' @export
 py_list_attributes <- function(x) {
   ensure_python_initialized()
+  if (py_is_module_proxy(x))
+    py_resolve_module_proxy(x)
```

Comment:
py_resolve_module_proxy may already do the equivalent of ensure_python_initialized (not sure, worth a check)